lc0_main.go

```diff
@@ -557,6 +569,9 @@ func nextGame(httpClient *http.Client, count int) error {
 					}
 				}
 				if err != nil || ng.Type != nextGame.Type || ng.Sha != nextGame.Sha {
```

Comment:
Its probably clearer to just do `if err!= nil` here (keeping the old code) and later test for the other conditions to do `pendingNextGame = &ng`